### PR TITLE
Issues/#245 8 Functional tests for stage, passing 52/75

### PR DESCRIPTION
**Pull request by ealefad**
Add tests:
```
p2p_invalid_block
p2p_invalid_tx
p2p_fingerprint
p2p_compactblocks
feature_versionbits_warning
```
Passing 52/75

### CODE REVIEW

File: eqbtest/functional/p2p_fingerprint.py [hunk 1]
```diff
@@ -75,14 +75,14 @@ def last_header_equals(self, expected_hash, node):
...
-        raise SkipTest("Disabled to make issues/#20-tx-structure pass")  # EQB_TODO: disabled test
+        #raise SkipTest("Disabled to make issues/#20-tx-structure pass")  # EQB_TODO: disabled test
         node0 = self.nodes[0].add_p2p_connection(P2PInterface())
 
         network_thread_start()
         node0.wait_for_verack()
 
         # Set node time to 60 days ago
-        self.nodes[0].setmocktime(int(time.time()) - 40 * 24 * 60 * 60)  # EQB_TODO: 40 days -> 60
+        self.nodes[0].setmocktime(int(time.time()) - 50 * 24 * 60 * 60)  # EQB_TODO: 50 days -> 60
```
Comment (macterra) [inline eqbtest/functional/p2p_fingerprint.py:85]: Comment incorrect?

**Review by macterra — APPROVED** [review]

File: eqbtest/functional/p2p_fingerprint.py (hunk 1 above)
Comment (ealefad) [reply eqbtest/functional/p2p_fingerprint.py:85]: I left this `TODO` as a reminder to change it back from 50 (today's actual value related to the genesis block time) to 60 days (original value) at the end of the next week, so the comment is correct.

Comment (ealefad) [reply eqbtest/functional/p2p_fingerprint.py:85]: ah, sorry, right, the comment above is incorrect

Comment (ealefad) [reply eqbtest/functional/p2p_fingerprint.py:85]: it'll be ok on the next week

**Pull request merged**